tests/lib/tools/tests.backup

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -e
+
```

Comment:
I think you need to add the license text here.

Reply:
sorry, which one? I don't see that in the other tools.